help/crypto.py

```diff
@@ -0,0 +1,13 @@
+from hashlib import sha256
+
+#TODO how to name that folder 
+#TODO  Maybe later try to use cyptography library, using key to hash password ?
+# https://www.mssqltips.com/sqlservertip/5173/encrypting-passwords-for-use-with-python-and-sql-server/
+
+
+def hash_password(password):
+    h = sha256()
```

Comment:
is h created and destroyed every time you call this method? if yes then its a waste and check how to construct that sha256() once

Reply:
done